lib/private/legacy/OC_User.php

```diff
@@ -172,7 +172,11 @@ public static function loginWithApache(\OCP\Authentication\IApacheBackend $backe
 				$userSession = \OC::$server->getUserSession();
 				$userSession->setLoginName($uid);
 				$request = OC::$server->getRequest();
-				$userSession->createSessionToken($request, $uid, $uid);
+				$password = null;
+				if ($backend instanceof OCP\Authentication\IProvideUserSecretBackend) {
```

Comment:
```suggestion
				if ($backend instanceof \OCP\Authentication\IProvideUserSecretBackend) {
```

Needs a trailing backslash as otherwise it would be considered to be within the current namespace, otherwise good from my side :+1: